src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,106 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * Representation of a storage file which can be read from and written to.
+ *
+ * @author Joshua Yong
+ */
+public class Storage {
+
+    private String filePath;
+
+    /**
+     * Class constructor.
+     *
+     * @param filePath The file path of the file to be used.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Parses the associated file, returning a collection of Tasks.
+     * @return An ArrayList of Tasks corresponding to the data in the file.
+     * @throws DukeException if the file is formatted incorrectly or cannot be found
```

Comment:
Perhaps you could capitalise the first letter of the sentence after the word DukeException? Same as above: 
```suggestion
     * @throws DukeException If the file is formatted incorrectly or cannot be found
```